sys-apps/mlocate/mlocate-0.26-r3.ebuild

```diff
@@ -56,7 +60,14 @@ src_install() {
 pkg_postinst() {
 	elog "The database for the locate command is generated daily by a cron job,"
 	elog "if you install for the first time you can run the updatedb command manually now."
+	elog "If you needn't cron job, please append the /etc/cron.daily/mlocate to INSTALL_MASK."
```

Comment:
Well, the message will be different. On new/upgraded installation we will tell users who are able to use the systemd timer that they can switch from cron job to systemd timer if they like. But we will take care of that on merge.